client/my-sites/themes/single-site.jsx

```diff
@@ -156,12 +157,22 @@ var ThemesSingleSite = React.createClass( {
 						site={ site }
 						canCustomize={ site && site.isCustomizable() } />
 				</CurrentThemeData>
-				<UpgradeNudge
-					title={ this.translate( 'Get Custom Design with Premium' ) }
-					message={ this.translate( 'Customize your theme using premium fonts, color palettes, and the CSS editor.' ) }
-					feature="custom-design"
-					event="themes_custom_design"
-				/>
+				{ ( abtest( 'themesHeaderNudge' ) === 'themes_custom_design' ) && (
+					<UpgradeNudge
+						title={ this.translate( 'Get Custom Design with Premium' ) }
+						message={ this.translate( 'Customize your theme using premium fonts, color palettes, and the CSS editor.' ) }
+						feature="custom-design"
+						event="themes_custom_design"
+					/>
+				) }
+				{ ( abtest( 'themesHeaderNudge' ) === 'themes_premium_theme_more_traffic' ) && (
+					<UpgradeNudge
+						title={ this.translate( 'Blogs with Premium theme get at least 7 times more traffic' ) }
```

Comment:
Well, our data indicates that's actually 30 times more, but that sounded less believable...